player/ucb1_tuned/ucb1_tuned_node.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+from player.mcts.mcts_node import MctsNode
+
+
+class Ucb1TunedNode(MctsNode):
+    @property
+    def variance_bound(self):
+        return self.reward_squared / self.number_of_runs - self.sample_mean ** 2
+    
+    
+    @property
+    def eval(self) -> float:
+        ratio_of_visits = np.sqrt(np.log(self.parent.number_of_runs) / self.number_of_runs)
+        return self.sample_mean + np.sqrt(ratio_of_visits * min(0.25, self.action_variance() + 2 * ratio_of_visits))
```

Comment:
What is `self.action_variance()` here? I guess you meant `variance_bound`? So probably `variance_bound` should be renamed to `action_variance`?

Reply:
yes ...

Reply:
Don't you worry :)